### PR TITLE
(feat) Add ability to end an active visit from the active visits table

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
https://user-images.githubusercontent.com/19533785/169823613-6cde21c7-2364-4fcb-9096-6c9677e29ba8.mp4

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by CynthiaKamau** [comment]
Kindly review @donaldkibet @denniskigen @makombe

File: packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx [hunk 1]
```diff
@@ -54,6 +54,13 @@ type FilterProps = {
 function ActionsMenu({ patientUuid }: { patientUuid: string }) {
   const { t } = useTranslation();
 
+  const openEndVisitModal = useCallback(() => {
```
Comment (denniskigen) [inline packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx:57]: Nit:

```suggestion
  const launchEndVisitModal = useCallback(() => {
```

File: packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx [hunk 2]
```diff
@@ -76,6 +83,7 @@ function ActionsMenu({ patientUuid }: { patientUuid: string }) {
       <OverflowMenuItem
         className={styles.menuItem}
         id="#endVisit"
+        onClick={openEndVisitModal}
```
Comment (denniskigen) [inline packages/esm-outpatient-app/src/active-visits/active-visits-table.component.tsx:86]:
```suggestion
        onClick={launchEndVisitModal}
```

**Review by denniskigen — COMMENTED** [review]
LGTM, save for a minor tweak.

**Review by denniskigen — APPROVED** [review]

**Pull request merged**